File: hw/fuentes/U2/HW12Cluster0/VinylRecords/src/main/java/ec/edu/espe/model/VinylRecord.java
package ec.edu.espe.model;
/**
 *
 * @author jorge
 */
public class VinylRecord {
    private String tittle;
    private String artist;
    private int year;
    private String gender;
    private String duration;
    private String label;
    private String country;
    private String color;
    private int trackCount;
    private String limitedEdition;

    public VinylRecord(String tittle, String artist, int year, String gender, String duration, String label, String country, String color, int trackCount, String limitedEdition) {
        this.tittle = tittle;
        this.artist = artist;
        this.year = year;
        this.gender = gender;
        this.duration = duration;
        this.label = label;
        this.country = country;
        this.color = color;
        this.trackCount = trackCount;
        this.limitedEdition = limitedEdition;
    }
    
    @Override
    public String toString(){
        return "VinylRecord{" + "title='" + tittle + '\'' + ", artist='" + artist + '\'' + ", year=" + year + ", gender='" + gender + '\'' + ", duration='" + duration + '\'' + ", label='" + label + '\'' + ", country='" + country + '\'' + ", color='" + color + '\'' + ", trackCount=" + trackCount + ", limitedEdition='" + limitedEdition + '\'' + '}';
    }
    
}

File: hw/fuentes/U2/HW12Cluster0/VinylRecordsPY/VinylRecord.py
class VinylRecord:
    def __init__(self, title, artist, year, gender, duration, label,
                 country, color, track_count, limited_edition):
        self.title = title
        self.artist = artist
        self.year = year
        self.gender = gender
        self.duration = duration
        self.label = label
        self.country = country
        self.color = color
        self.track_count = track_count
        self.limited_edition = limited_edition

    def to_dict(self):
        return {
            "title": self.title,
            "artist": self.artist,
            "year": self.year,
            "gender": self.gender,
            "duration": self.duration,
            "label": self.label,
            "country": self.country,
            "color": self.color,
            "track_count": self.track_count,
            "limited_edition": self.limited_edition
        }
